substrate/frame/root-testing/src/lib.rs

```diff
@@ -50,5 +59,13 @@ pub mod pallet {
 			ensure_root(origin)?;
 			Ok(())
 		}
+
+		#[pallet::call_index(1)]
+		#[pallet::weight(0)]
+		pub fn trigger_defensive(origin: OriginFor<T>) -> DispatchResult {
+			ensure_root(origin)?;
+			Self::deposit_event(Event::DefensiveTestCall);
```

Comment:
Oh, you also have to actually trigger the defensive logic via `frame_support::defensive!("Howdy");`. or something like this.

Reply:
@ggwpez I've added the `frame_support::defensive!` logic. What do you think about the message?